package.json

```diff
@@ -1,10 +1,11 @@
 {
   "name": "json-statham",
-  "version": "1.6.0",
+  "version": "1.6.1",
   "description": "Kick your JSON's ass, with json-statham's help.",
   "main": "index.js",
   "scripts": {
-    "test": "eslint index.js lib && mocha -b"
+    "test": "eslint index.js lib && mocha -b",
+    "build": ".\\node_modules\\.bin\\buble -i index.js -o dist/index.js && .\\node_modules\\.bin\\buble -i lib/ -o dist/lib/"
```

Comment:
You don't need the prefix, it should be on the path already.

`buble -i index.js -o dist/index.js && buble -i lib/ -o dist/lib/`